### PR TITLE
[zh-cn] Update MathML Attribute page

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

Fix up translation in mathml attribute page.

### Related issues and pull requests

part of #14272

### CODE REVIEW

File: files/zh-cn/web/mathml/attribute/index.md [hunk 1]
```diff
@@ -1,102 +1,197 @@
...
-此页面是 MathML 属性的字母表。每个属性的更多详细信息请看这里[element pages](/zh-CN/docs/MathML/Element)。
+{{MathMLRef}}
 
-注意：
+此页面是 MathML 属性的字母表。要了解每个属性的更多详细信息，请参见 [MathML 元素页面](/zh-CN/docs/Web/MathML/Element)和[全局属性页面](/zh-CN/docs/Web/MathML/Global_attributes)。[值](/zh-CN/docs/Web/MathML/Values)页面也介绍了有关 MathML 属性常用值的一些说明。
 
-- MathML 的 {{ MathMLElement("mstyle") }} 和 {{ MathMLElement("math") }} 元素接受所有 MathML 的描述元素。
-- 请参阅 MathML 中值（[values](/zh-CN/docs/MathML/Attributes/Values)）和单位的注释值。
+> **备注：** 就像 [MathML](/zh-CN/docs/Web/MathML) 主页所解释的那样，MDN 使用 [MathML 核心](https://w3c.github.io/mathml-core/)作为参考规范。不过，一些浏览器仍在使用的传统功能也被记录在案。你可以在 
+ [MathML 4](https://w3c.github.io/mathml/) 中找到有关这些功能和其他功能的更多详细信息。
```
Comment (jasonren0403) [inline files/zh-cn/web/mathml/attribute/index.md:11]:
```suggestion
> **备注：** 就像 [MathML](/zh-CN/docs/Web/MathML) 主页所解释的那样，MDN 使用 [MathML 核心](https://w3c.github.io/mathml-core/)作为参考规范。不过，一些浏览器仍在使用的传统功能也被记录在案。你可以在 [MathML 4](https://w3c.github.io/mathml/) 中找到有关这些功能和其他功能的更多详细信息。
```

File: files/zh-cn/web/mathml/attribute/index.md [hunk 2]
```diff
@@ -1,102 +1,196 @@
...
 
-此页面是 MathML 属性的字母表。每个属性的更多详细信息请看这里[element pages](/zh-CN/docs/MathML/Element)。
+{{MathMLRef}}
 
-注意：
+此页面是 MathML 属性的字母表。要了解每个属性的更多详细信息，请参见 [MathML 元素页面](/zh-CN/docs/Web/MathML/Element)和[全局属性页面](/zh-CN/docs/Web/MathML/Global_attributes)。[值](/zh-CN/docs/Web/MathML/Values)页面也介绍了有关 MathML 属性常用值的一些说明。
 
-- MathML 的 {{ MathMLElement("mstyle") }} 和 {{ MathMLElement("math") }} 元素接受所有 MathML 的描述元素。
-- 请参阅 MathML 中值（[values](/zh-CN/docs/MathML/Attributes/Values)）和单位的注释值。
+> **备注：** 就像 [MathML](/zh-CN/docs/Web/MathML) 主页所解释的那样，MDN 使用 [MathML 核心](https://w3c.github.io/mathml-core/)作为参考规范。不过，一些浏览器仍在使用的传统功能也被记录在案。你可以在 [MathML 4](https://w3c.github.io/mathml/) 中找到有关这些功能和其他功能的更多详细信息。
```
Comment (yin1999) [inline files/zh-cn/web/mathml/attribute/index.md:10]:
```suggestion
> **备注：** 就像 [MathML](/zh-CN/docs/Web/MathML) 主页所解释的那样，MDN 使用 [MathML 核心](https://w3c.github.io/mathml-core/)作为参考规范。不过，一些浏览器仍在使用的遗留特性也被记录在案。你可以在 [MathML 4](https://w3c.github.io/mathml/) 中找到有关这些特性和其他特性的更多详细信息。
```

File: files/zh-cn/web/mathml/attribute/index.md [hunk 3]
```diff
@@ -1,102 +1,196 @@
...
+      </td>
+      <td>
+        指定表格相对于环境的垂直对齐方式。
+      </td>
+    </tr>
+    <tr>
+      <td><code>background</code> {{deprecated_inline}}</td>
+      <td>{{ MathMLElement("mstyle") }}</td>
+      <td>
+        请使用 CSS <a href="/zh-CN/docs/Web/CSS/background-color"><code>background-color</code></a> 属性。
```
Comment (yin1999) [inline files/zh-cn/web/mathml/attribute/index.md:66]:
```suggestion
        请使用 CSS <a href="/zh-CN/docs/Web/CSS/background-color"><code>background-color</code></a> 代替。
```

File: files/zh-cn/web/mathml/attribute/index.md [hunk 4]
```diff
@@ -1,102 +1,196 @@
...
+      </td>
+    </tr>
+    <tr>
+      <td><code>close</code>  {{deprecated_inline}}</td>
+      <td>{{ MathMLElement("mfenced") }}</td>
+      <td>用于结尾分隔符的字符串。</td>
+    <tr>
```
Comment (yin1999) [inline files/zh-cn/web/mathml/attribute/index.md:73]: Missing the `color` and some other attributes here

File: files/zh-cn/web/mathml/attribute/index.md [hunk 5]
```diff
@@ -1,102 +1,196 @@
...
+      <td><code>voffset</code></td>
+      <td>{{ MathMLElement("mpadded") }}</td>
+      <td>一个 <a href="/zh-CN/docs/Web/CSS/length-percentage"><code>&lt;length-percentage&gt;</code></a> 值，表示子内容的定位点相对于元素定位点的垂直位置。
+</td>
+    </tr>
+    <tr>
+      <td><code>width</code></td>
+      <td>
+        {{ MathMLElement("mpadded") }}、{{ MathMLElement("mspace") }}】{{ MathMLElement("mtable") }}
```
Comment (yin1999) [inline files/zh-cn/web/mathml/attribute/index.md:178]:
```suggestion
        {{ MathMLElement("mpadded") }}、{{ MathMLElement("mspace") }}、{{ MathMLElement("mtable") }}
```

**Review by yin1999 — APPROVED** [review]
Thanks

**Pull request merged**